.github/workflows/main.yml

```diff
@@ -53,12 +53,11 @@ jobs:
           python setup.py bdist_wheel --skip-build
 
       - uses: actions/upload-artifact@v3
+        # Upload artefacts even if tests fail
```

Comment:
typo: artifacts.

Reply:
oops, my french coming through ^^" (I think technically both are correct spelling depending on the culture, but yeah, let's be consistent with the action name)